ci/test_cpp.sh

```diff
@@ -32,9 +32,9 @@ trap "EXITCODE=1" ERR
 set +e
 
 rapids-logger "Running googletests"
-for gt in "$CONDA_PREFIX/bin/gtests/librmm/"* ; do
-    ${gt} --gtest_output=xml:${RAPIDS_TESTS_DIR}/
-done
+cd $CONDA_PREFIX/bin/gtests/librmm/
+export GTEST_OUTPUT=xml:${RAPIDS_TESTS_DIR}/
+ctest -j20 --output-on-failure
```

Comment:
FWIW we typically set parallelism in Python tests more in the 5-10 range, but since this is working for librmm tests right now happy to stick with the bigger number. These tests seem to have much lower average GPU utilization.

Reply:
I would guess that `-j20` is only really using 20 parallel test invocations for the default tests (which default to requesting 5% of the GPU). If one of the larger tests requiring 60% of the GPU is executed, I think it will probably run significantly less than 20 jobs at the same time. tl;dr It's probably safe to have a large value here for parallelism as long as the bigger tests are marked with a larger resource request.